nodered.json

```
@@ -22,8 +22,8 @@
                 }
             }
         },
-            "version": "latest",
-        "description": "Low-code programming for event-driven applications. Node-RED is a programming tool for wiring together hardware devices, APIs and online services in new and interesting ways. <p>Based on the official Node-RED docker image: <a href='https://hub.docker.com/r/nodered/node-red' target='_blank'>https://hub.docker.com/r/nodered/node-red</a>.",
+        "version": "1.1",
+        "description": "Low-code programming for event-driven applications. Node-RED is a programming tool for wiring together hardware devices, APIs and online services in new and interesting ways. <p>Based on the official docker image: <a href='https://hub.docker.com/r/nodered/node-red' target='_blank'>https://hub.docker.com/r/nodered/node-red</a>, available for amd64 architecture only.",

```

```suggestion
        "description": "Low-code programming for event-driven applications. Node-RED is a programming tool for wiring together hardware devices, APIs and online services in new and interesting ways. <p>Based on the official docker image: <a href='https://hub.docker.com/r/nodered/node-red' target='_blank'>https://hub.docker.com/r/nodered/node-red</a>, available for amd64 and arm64 architecture.",

```

@Hooverdan96 there is a dropdown on this one at Docker hub and it looks like there is, at least when I checked just now, an arm64 option so I've popped in the text accordingly.